packages/demos-babylon/.gitignore

```diff
@@ -0,0 +1 @@
+src/common/demo-credentials.js
```

Comment:
I was experimenting with package-level ignores. I'm not sure what the best practice is when it comes to monorepos, but it seemed useful for each package to "own" the definition of what files should be ignored instead of having a top-level uber ignore. But I don't feel strongly about this and would be happy to change. LMK what your preference is.

Reply:
I don't really have a preference. Another idea is to put `demo-credentials.js` in the root .gitignore, and then this file will get ignored in all repos.  up to you!

Reply:
For now I will rename "demo-credentials.template.js" to "demo-credentials.js" to allow test to pass without requiring user intervention. This means we'll no longer be protecting against checking in Cognito IDs, but we'll look for a different fix for that in a future PR.